src/components/DateRangeInput.tsx

```diff
@@ -37,9 +29,11 @@ export default function DateRangeInput({
     const yearDiff = value as number;
     const key = now.year - yearDiff;
     shortcuts[key] = {
-      text: `End of ${now.year - yearDiff}`,
+      text: `Year ${now.year - yearDiff}`,
```

Comment:
the shortcuts are now yearly (useful for seeing a summary of whole previous years)